github3/pulls.py

```diff
@@ -46,6 +46,7 @@ def __init__(self, dest, direction):
         if dest.get('repo'):
             self._repo_name = dest['repo'].get('name')
             self._repo_owner = dest['repo']['owner'].get('login')
+            self.repository = Repository(self.repository, self)
```

Comment:
So, I'm not sure how well you know Python, but if you look at this file there's nothing defined as `Repository`. This is why the [tests](https://travis-ci.org/sigmavirus24/github3.py/jobs/145266794#L212) are failing. There is a definition for `Repository` in `github3.repos.repo` though and we can use that by importing it. This, however, is tricky because that module imports this one. That means we have to do the import locally. So you need to add one more line before this one:

```py
from .repos.repo import Repository
```

This will allow this line to work.

Let me know when you've updated this so I can merge it!